src/converters/xml.cr

```diff
@@ -0,0 +1,88 @@
+module OQ::Converters::Xml
+  @@at_root : Bool = true
+
+  def self.deserialize(input : IO, output : IO, **args) : Nil
+    raise "Not Implemented"
+  end
+
+  def self.serialize(input : IO, output : IO, **args) : Nil
+    json = JSON::PullParser.new(input)
+    builder = XML::Builder.new(output)
+    indent, prolog, root, xml_item = self.parse_args(args)
+
+    builder.indent = indent
+
+    builder.start_document "1.0", "UTF-8" if prolog
+    builder.start_element root unless root.blank?
+
+    loop do
+      emit builder, json, xml_item: xml_item
+      break if json.kind == :EOF
+    end
+
+    builder.end_element unless root.blank?
+    builder.end_document if prolog
+    builder.flush unless prolog
+  end
+
+  private def self.parse_args(args : NamedTuple) : Tuple(String, Bool, String, String)
+    {
+      args["indent"],
```

Comment:
Oh, wow, I didn't remember you could access a named tuple with a string, I though only symbol was valid.